docs/source/tutorials/pasha/pasha.rst

```diff
@@ -0,0 +1,74 @@
+The experimental comparison shows that PASHA identifies well-performing hyperparameter
+configurations and architectures while consuming significantly fewer computational
+resources than ASHA.
+
+What is PASHA?
+--------------------------------------------
+
+The goal of `PASHA <https://openreview.net/forum?id=syfgJE6nFRW>`_ is to identify
+well-performing configurations significantly faster than current methods,
+so that we can then retrain the model with the selected configuration
+(in practice on the combined training and validation sets). By giving preference
+to evaluating more configurations rather than evaluating them for longer than
+needed, PASHA can lead to significant speedups while achieving similar performance
+as existing methods.
+
+PASHA is a variant of ASHA that starts with a small amount of initial resources
+and gradually increases them depending on the stability of configuration rankings
+in the top two rungs (rounds of promotion). Each time the ranking of configurations
+in the top two rungs becomes inconsistent, PASHA increases the maximum number
+of resources. This can be understood as "unlocking" a new rung level. An illustration
+of how PASHA stops early if the ranking of configurations has stabilized
+is shown in Figure 1.
+
+.. image:: img/pasha_illustration.png
+   :width: 768 px
+
+Given that deep-learning algorithms typically rely on stochastic gradient descent, ranking 
+inconsistencies can occur between similarly performing configurations. Hence, we need some 
+benevolence in estimating the ranking. As a solution, PASHA uses a soft-ranking
+approach where we group configurations based on their validation performance metric
+(e.g. accuracy).
+
+In soft ranking, configurations are still sorted by predictive performance but they
+are considered equivalent if the performance difference is smaller than a
+value :math:`\epsilon` (or equal to it). Instead of producing a sorted list of configurations,
+this provides a list of lists where for every position of the ranking there is a list
+of equivalent configurations. The concept is explained graphically in Figure 2. 
+The value of :math:`\epsilon` is automatically estimated by measuring noise in rankings.
+
+.. image:: img/soft_ranking.png
+   :width: 768 px
+
+How well does PASHA work?
+--------------------------------------------
+
+Experimental evaluation has shown PASHA consistently leads to strong improvements in runtime,
+while achieving similar accuracies as ASHA. PASHA is e.g. three times faster than ASHA on NASBench201.
+Full experiments and further details are available in 
+`PASHA: Efficient HPO and NAS with Progressive Resource Allocation <https://openreview.net/forum?id=syfgJE6nFRW>`_.
+
+Launcher script
+--------------------------------------------
+
+We provide an example script 
+`launch_pasha_nasbench201.py <https://github.com/awslabs/syne-tune/blob/main/examples/launch_pasha_nasbench201.py>`_
```

Comment:
We do this a little differently, and it would be great if you could do this:
- Add section in `docs/source/examples.rst` for this example. Just have a look at the `launch_nasbench201_simulated` example how this looks like
- Refer to the example in this file from here

Reply:
thanks, I've modified the corresponding parts accordingly in a new commit